styles/keybr.com/catppuccin.user.css

```
@@ -0,0 +1,131 @@
+/* ==UserStyle==
+@name keybr.com Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/keybr.com
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/keybr.com
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/keybr.com/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Akeybr.com
+@description Soothing pastel theme for keybr.com
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('keybr.com') {
+  html[data-color="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  html[data-color="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
```

You should support `[data-color="system"]` as well:
```suggestion
  html[data-color="dark"] {
    #catppuccin(@darkFlavor, @accentColor);
  }
  html[data-color="light"] {
    #catppuccin(@lightFlavor, @accentColor);
  }
  html[data-color="system"] {
    @media (prefers-color-scheme: light) {
	  #catppuccin(@lightFlavor, @accentColor);
    }
    @media (prefers-color-scheme: dark) {
	  #catppuccin(@darkFlavor, @accentColor);
    }
  }
```